code/obj/item/cigarette.dm

```diff
@@ -288,6 +447,8 @@
 				return
 			else
 				src.put_out(M, "<span style='color:red'><b>[M]</b>'s [src.name] goes out.</span>")
+				if (istype(src.loc, /obj/item/clothing/mask/cig_bundle))
+					src.set_loc(get_turf(src.loc))
```

Comment:
Might be worth updating the cig_bundle's sprite here?

Reply:
Ah good catch ty.